### ISSUE TITLE
Кодекс поведения v1.4 

### ISSUE THREAD

**Issue by tabuna** [LaravelRUS/internals#1]
Есть более свежая версия http://contributor-covenant.org/version/1/4/ru/

**Comment by vladimir-yuldashev** [comment]
Не заметил, обновлю сегодня.

**Comment by tabuna** [comment]
Так же очень не плохие кодексы есть у linux диструбутивов

**Comment by SerafimArts** [comment]
Что это? =)

Если про правила чатиков, они уже давно согласованы и лежат тут: https://gitter.im/LaravelRUS/topics/topic/582f01c2004785313cbfb669/devyat-krugov-chata?source=orgpage

**Comment by vladimir-yuldashev** [comment]
Это нормы поведения в сообществе, не только в чате. А в том формате, которые оформлены правила чата, что ты прислал (кури доки, ты пых сначала выучи) - неприемлемы.

**Comment by SerafimArts** [comment]
@vladimir-yuldashev и где ты был, когда их составляли? Если что-то не нравится - редактируй (права у тебя есть). Любые фатальные изменения стоит согласовывать с администрацией всех чатиков, где они приняты (русское сообщество vuejs и laravel, т.е. наше).

И пожалуйста, для тех кто в танке, поясните что за движуху вы тут внезапно устроили =))))

**Comment by vladimir-yuldashev** [comment]
@SerafimArts движуха началась в связи с последними событиями в Telegram-чате. 

Правила, которые ты скинул находятся в Gitter (который кмк пропащий во многих смыслах), пускай на него и распространяются. Эти правила общие для сообщества.

**Comment by SerafimArts** [comment]
> Правила, которые ты скинул, находятся в Gitter

Нет, они находятся на сайте (http://laravel.su/articles/nine-circles-of-chat)

На счёт формальности - ты прав. В этом случае, пожалуйста, либо учитывай пожелания участников более активного чата (т.е. в гиттере, где 4к+ участников, против 400 в телеграмме) при разработке нового свода, переводя определения "заворачивать ссылки на ютуб" (и аналогичные) в формальный язык. Либо явно подчеркни, что это касается только Телеграмма, тогда мы займёмся адаптацией оных под гиттер.

А так, всё ок, правильное дело делаете, давно надо было подчистить основные правила =)

**Comment by VladShcherbin** [comment]
Было бы круто эти правила сделать и для gittera ибо количество быдло-троллинга и чсв там зашкаливает.

**Comment by vladimir-yuldashev** [comment]
@tabuna Обновил.

**Comment by vladimir-yuldashev** [comment]
@SerafimArts

печально, что такое попадает на сайт. я скинул ссылку на этот репозиторий в основное сообществе ВК и все могут сюда зайти и предложить свои изменения, а так же высказаться. Этот кодекс достаточно универсальный и адаптации не требует. 

Для таких правил как "оформления кода в теги" можно создать документ `gitter.md` т.к. в различных мессенджерах формат действительно различается.

**Comment by jhaoda** [comment]
> Правила, которые ты скинул, находятся в Gitter (который кмк пропащий во многих смыслах), пускай на него и распространяются.

Ну всё, Владимир разрешил, а на чат в телеге мне насрать...

**Comment by jhaoda** [comment]
@vladimir-yuldashev а что здесь-то не ответишь,а? Я никогда не был в телеграмском чате и не собираюсь, почему же мне не должно быть на него насрать?

**Comment by vladimir-yuldashev** [comment]
@jhaoda какой ответ от меня ожидается? я не собираюсь продолжать диалог, когда он начался с неуважения.

**Comment by jhaoda** [comment]
@vladimir-yuldashev неуважения к чему? К тому, чего я никогда не видел?

**Comment by vladimir-yuldashev** [comment]
@jhaoda неуважительное общение.

**Comment by jhaoda** [comment]
@vladimir-yuldashev в чем ты увидел неуважение?

**Comment by vladimir-yuldashev** [comment]
@jhaoda слово "[насрать](https://ru.wiktionary.org/wiki/насрать)" является непристойным и оскорбительным.

**Comment by jhaoda** [comment]
Вот это приехали...
